src/pfe/file-watcher/server/src/controllers/projectsController.ts

```diff
@@ -58,7 +58,15 @@ const BUILD_KEY = "projectStatusController.buildRank";
 // timeout to ping build projects
 setInterval(checkBuildQueue, 5000);
 
-const availabilityTimeout = 5000;
+export async function removeProjectFromRunningBuildQueue(projectID: string): Promise<Array<BuildQueueType>> {
```

Comment:
Perhaps right a comment above the function saying its only for test, since this is exported; ppl may use it from other modules without reading the log comment

And maybe update the log statement saying we're removing the project from the queue for debugging purposes

Reply:
Done.